python/py-coremltools/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-coremltools
+version             5.2.0
+license             BSD
+maintainers         nomaintainer
+supported_archs     arm64 x86_64
```

Comment:
this doesn't work on i386? Also if appears you'd need C++17 (```set(CMAKE_CXX_STANDARD 17)```)

Reply:
Thanks done.